pkg/operator/ceph/cluster/osd/osd_test.go

```diff
@@ -170,10 +196,16 @@ func TestAddNodeFailure(t *testing.T) {
 
 	// create a fake clientset that will return an error when the operator tries to create a replica set
```

Comment:
This comment needs to be updated to say `job` instead of `replica set`.

Reply:
+1